### PR TITLE
Add support for IntelliJ IDEA

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ const editors = () => [{
 	paths: [],
 	keywords: []
 }, {
+	id: 'intellij',
+	name: 'IntelliJ',
+	bin: 'idea',
+	paths: [],
```
Comment (sindresorhus) [inline index.js]: Can you add the path to the binary?

File: readme.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ This module is used by [`open-editor`](https://github.com/sindresorhus/open-edit
 - Atom
 - Visual Studio Code
 - WebStorm
+- IntelliJ
```
Comment (sindresorhus) [inline readme.md]: Place it at the bottom. Same with the other places.

File: index.js [hunk 3]
```diff
@@ -35,6 +35,12 @@ const editors = () => [{
...
+	keywords: []
```
Comment (sindresorhus) [inline index.js]: Keywords could be: idea, java, jetbrains, ide

File: index.js [hunk 4]
```diff
@@ -35,6 +35,12 @@ const editors = () => [{
 	paths: [],
 	keywords: []
 }, {
+	id: 'intellij',
+	name: 'IntelliJ',
```
Comment (sindresorhus) [inline index.js]: Seems the official name is: IntelliJ IDEA

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (SimenB) [reply index.js]: The `idea` binary, or the app?
The app itself varies with version (mine is now `RUN_PATH = u'/Users/simen/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/171.4424.14/IntelliJ IDEA 2017.1 EAP.app'`).

The binary is generated by hand in the IDE, with a user-defined path. This is the default:

image

Should I use that?

This is the exact same as webstorm, btw. WebStorm is just the JS parts of intellij, so changes to one here should change the other

File: index.js (hunk 3 above)
Comment (SimenB) [reply index.js]: Added 😄

File: readme.md (hunk 2 above)
Comment (SimenB) [reply readme.md]: Done

File: index.js (hunk 1 above)
Comment (sindresorhus) [reply index.js]: Nah, it's fine. That's a standard location to look.

**Review by sindresorhus — APPROVED** [review]

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**